core/templates/pages/topic-viewer-page/stories-list/topic-viewer-stories-list.directive.html

```diff
@@ -18,9 +18,9 @@
        ng-swipe-left="scroll(false)">
     <div class="oppia-topic-viewer-carousel-tiles">
       <md-card ng-repeat="canonicalStory in canonicalStories" class="oppia-story-summary-tile">
-        <story-summary-tile story-id="canonicalStory.id"
-                            title="canonicalStory.title"
-                            description="canonicalStory.description">
+        <story-summary-tile story-id="canonicalStory.getId()"
```

Comment:
Can you please add a screenshot of the UI after this change?

Reply:
![image](https://user-images.githubusercontent.com/32016324/79643779-2acdf280-81c2-11ea-97ef-ca331d4dac99.png)

Reply:
console for code confirmation

Reply:
Thanks for checking!